scripts/deploy.sh

```diff
@@ -48,8 +49,13 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   if [ -n "${DATA_DISK_DEVICE}" ]; then
     export BOOTSTRAP_OPTS="${BOOTSTRAP_OPTS} bootstrap_host_data_disk_device=${DATA_DISK_DEVICE}"
   fi
+
   # force the deployment of haproxy for an AIO
   export DEPLOY_HAPROXY="yes"
+
+  # Enable host security hardening for AIO builds
+  export DEPLOY_HARDENING="yes"
```

Comment:
I can certainly understand that, but we are currently testing upstream Mitaka/Newton OSA AIO's with security hardening enabled. I've also proposed that for OSA's Liberty release as well.  The role should not impact a running RPC-O deployment, and if it does, we've found a bug.

Are there any specific concerns about the content of the role and applying those configurations to an AIO?